packages/compass-preferences-model/src/feature-flags.ts

```diff
@@ -21,6 +21,7 @@ export type FeatureFlags = {
   enableQueryHistoryAutocomplete: boolean;
   enableProxySupport: boolean;
   enableRollingIndexes: boolean;
+  enableGlobalWrites: boolean;
```

Comment:
Why are we adding it as a feature flag on compass side? should it not be part of preferences and FF as part of MMS only?

Reply:
Good question. Tbh I am confused about the different types of preferences in compass and wanted to check this with @gribnoysup. I think Global Writes should be in the same group as Rolling Indexes, as both features are web only for now, but we might be adding them to desktop later.

Reply:
Yeah, so I think it's understandable that you can see these in either of those preferences groups, we don't really have a very clean separation there for case like this.

Saying that I think these map more to what we use the feature flag group of preferences for: they are temporary, will only be used for the initial rollout and then removed, we don't really need a dedicated preference for these features as their availability in UI has a stricter check against a cluster metadata that is not available in compass desktop (and at the point where it will be, they should start working without the feature flag required to toggle them)

This is different from something like saved queries feature where bringing it to the web will require more effort in making the system work universally across environments and so it's not something that is just temporarily disabled in the same sense as these that will be used for the rollout.

And to be clear this is a very loose definition that can be understandably argued against, we might consider introducing some new preferences group to remove the ambiguity, maybe something to think through a bit more, but probably doesn't need to block this particular change.

Reply:
my only concern was if this is a ff on compass side, it will show up in settings-modal. but not a blocker.